WeakAurasOptions/RegionOptions/DynamicGroup.lua

```diff
@@ -139,6 +139,23 @@ local function createOptions(id, data)
         OptionsPrivate.ResetMoverSizer()
       end,
     },
+    growEvents = {
```

Comment:
this (and the sortEvents option below) are new toys for custom sort code. Give it a comma separated list of state keys, and weakauras will try to only run your sort/grow code if those particular state values change.

The special key `changed` counts as a wildcard here and essentially re-enables existing behavior.


we may want to do a migration to set this value to "changed" for existing auras, as i think there may be some auras that will break with this change

Reply:
P.S. grow code always runs if sort code runs, and sort code always runs if a region is added or deleted. in a lot of cases, that will be the only time that we actually needed to relayout the dgroup